etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/SendOrderUseCase.java

```diff
@@ -27,8 +27,19 @@ public static SendOrderUseCase getInstance() {
     public void convertAndSend(final Order<?> order, String submissionId)
             throws UnableToSendOrderException {
 
+        var partnerMetadata =
+                new PartnerMetadata(
+                        "uniqueId",
+                        "senderName",
+                        "receiverName",
+                        Instant.now(),
+                        "abcd"); // For testing, delete when done
         try {
+            logger.logInfo("Trying to save the metadata");
             savePartnerMetadata(submissionId);
+            logger.logInfo("Trying to read the metadata");
+            partnerMetadataStorage.readMetadata(partnerMetadata.uniqueId());
+
```

Comment:
Just adding back the call to `savePartnerMetadata` isn't correct.  The merge was bad and the real fix is just to remove everything except the start of the `try` block and the call to `savePartnerMetadata`.  E.g....

```suggestion
        try {
            savePartnerMetadata(submissionId);
```

Reply:
Yes, totally agree. We are still using some of that code for testing purposes. We will delete that code before taking it out of draft.